scripts/test/runtests.ps1

```diff
@@ -5,65 +5,79 @@
 
 . "$PSScriptRoot\..\common\_common.ps1"
 
-$TestBinRoot = "$RepoRoot\artifacts\tests"
+$failCount = 0
+
+# Capture PATH for later
+$StartPath = $env:PATH
```

Comment:
We could. This was a tactical fix to get the tests to work (since they were basically just testing the stage 0 binaries previously), I didn't want to go too far, since I was just trying to get the build working :).